be/src/exec/json_parser.h

```diff
@@ -67,6 +69,13 @@ class JsonDocumentStreamParser : public JsonParser {
     simdjson::ondemand::object _curr;
     // _curr_ready denotes whether the _curr has been parsed.
     bool _curr_ready = false;
+    // _last_begin_offsets represent begin offet of last success object in _doc_stream
+    size_t _last_begin_offsets = 0;
+    // _batch_size using in batch mode parsing
+    size_t _batch_size = (config::json_parse_many_batch_size > 0) ?
```

Comment:
shall put this non-trivial initialization in constructor.

Reply:
updated